src/passworddisplaypanel.cpp

```
@@ -39,7 +39,11 @@ PasswordDisplayPanel::PasswordDisplayPanel(QGridLayout *grid,
 void PasswordDisplayPanel::clear() {
   while (m_grid->count() > 0) {
     QLayoutItem *item = m_grid->takeAt(0);
-    delete item->widget();
+    if (item->widget()) {
```

Clang (14.0.6)

[error] repeated branch in conditional chain

(bugprone-branch-clone,-warnings-as-errors)